acceptance/tests/load_libfacter.rb

```diff
@@ -1,31 +1,44 @@
-test_name 'Ruby can load libfacter'
+test_name 'C100161: Ruby can load libfacter' do
```

Comment:
So, wouldn't that be validated by testing that `facter` is in the $LOAD_PATH for the puppetlabs vendored ruby? If so, then most of the code in this test is not necessary.
```
step 'Ensure that ruby can load facter from the $LOAD_PATH' do
  on(agent, ruby_command(agent), '-e', '"require \'facter\'"')
end
```
What am I missing?